code-of-conduct.md

```diff
@@ -0,0 +1,66 @@
+---
+id: code-of-conduct
+title: Code of Conduct
+description: "Camunda Cloud Documentation Code of Conduct"
+keywords: ["code of conduct", "CoC", "community standards"]
+---
```

Comment:
Since this is a plain markdown page I don't think we need this section anymore.

Reply:
Removed in latest commit! :)